cli.js

```diff
@@ -86,7 +139,18 @@ module.exports = async function runCli () {
     const options = Object.assign({
       mochaOptions,
       lassoConfig,
-      lassoDependencies
+      lassoDependencies,
+      launchOptions: {
```

Comment:
Let's change this to `puppeteerLaunchOptions`. Although it is a bit more verbose, it's pretty obvious what the options are used for. It will also match up nicely with the rest of the options that can be supplied in the `.mocha-puppeteer-config.js` file (to be handled in a separate PR).